eventing/samples/gcp-pubsub-source/README.md

```diff
@@ -80,60 +86,45 @@ source is most useful as a bridge from other GCP services, such as
        name: in-memory-channel
    ```
 
-   If you're in the samples directory, you can also apply the `channel.yaml`
-   file:
+   If you're in the samples directory, you can apply the `channel.yaml` file:
 
    ```shell
    kubectl apply --filename channel.yaml
    ```
 
-1. Create a GCP PubSub Topic. If you change this, you also need to update the
-   `topic` in the `gcp-pubsub-source.yaml` file.
+1. Create a GCP PubSub Topic. If you change its name (`testing`), you also need to update the
+   `topic` in the [`gcp-pubsub-source.yaml`](./gcp-pubsub-source.yaml) file:
 
    ```shell
    gcloud pubsub topics create testing
    ```
 
 1. Replace the
    [`MY_GCP_PROJECT` placeholder](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
-   in `gcp-pubsub-source.yaml`. You should end up with a file which looks like
-   this:
+   in [`gcp-pubsub-source.yaml`](./gcp-pubsub-source.yaml) and apply it.
 
-   ```yaml
```

Comment:
I think the tech writing team is encouraging a style where short and medium configuration and code snippets are present in-line to improve the reading experience (at the cost of some maintenance for now).